src/service/action-impl.js

```diff
@@ -457,6 +469,55 @@ export function parseActionMap(s, context) {
 }
 
 /**
+ * Returns a function that generates a method argument value for a given token.
+ * The function takes a single object argument `data`.
+ * If the token is an identifier `foo`, the function returns `data[foo]`.
+ * Otherwise, the function returns the token value.
+ * @param {!{type: TokenType, value: *}} token
+ * @return {?function(!Object):string}
+ * @private
+ */
+function getActionInfoArgValue(token) {
+  const value = token.value;
+  if (token.type == TokenType.LITERAL) {
+    return () => value;
+  } else if (token.type == TokenType.ID) {
```

Comment:
So, this changes the spec in described in the https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#on

Looks like the docs should be changed. How compatible are we with the current doc?

Looks like we fallback to the literal value. Would that look strange per doc?